app/scheduler.py

```diff
@@ -569,19 +569,33 @@ def weekly_remainder():
                                     "data": None,
                                     "message_type" : "button_message",
                                     "message_key": "automated_weekly_less",
-                                    "url_link": "http://tms.excellencetechnologies.in/#/app/automateWeekly",
-                                    "button_text":"Submit an automatic weekly report"
-                                    }
+                                    "button":{"action":
+                                    [
+                                        {
+                                                "type": "button",
+                                                "text": "Submit an automatic weekly report",
+                                                "url": "http://tms.excellencetechnologies.in/#/app/automateWeekly"
```

Comment:
why is this hard coded here?

Reply:
sir this for TMS specific

Reply:
sir this is TMS branch not of notification system TMS git in short